.gitignore

```diff
@@ -1,2 +1,3 @@
-.idea
-vendor
+.idea/
+/vendor
+composer.lock
```

Comment:
We want to leave the composer.lock file in. Since it prevents unexpected build breaks.